symplyphysics/laws/electricity/circuits/resistance_of_the_serial_resistor_coil_capacitor_circuit.py

```diff
@@ -0,0 +1,69 @@
+from sympy import (Eq, solve, sqrt, I)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output,)
+from symplyphysics.core.expr_comparisons import expr_equals
+
+from symplyphysics.definitions import impedance_is_resistance_and_reactance as impedance_law
+from symplyphysics.laws.electricity.circuits import resistivity_of_serial_resistors as serial_law
+
+# Description
+## Consider an electrical circuit consisting of a capacitor, coil, and resistor connected in series.
+## Then you can find the total resistance of such a circuit.
+
+## Law is: R = sqrt(R0^2 + (Xl - Xc)^2), where
```

Comment:
I'm not sure it is really true. You can obtain absolute value of impedance but I am not sure you can name it `total resistance of a circuit`.

Reply:
> I'm not sure it is really true. You can obtain absolute value of impedance but I am not sure you can name it `total resistance of a circuit`.

This is what they call reactance here:
https://en.wikipedia.org/wiki/Electrical_impedance#:~:text=The%20total%20reactance%20is%20given%20by

Reply:
No, reactance is imaginary part of impedance.